docker-compose.cms.yml

```diff
@@ -0,0 +1,19 @@
+services:  
+  cms:
+    container_name: keystone-cms
+    build:
+      context: https://github.com/USSF-ORBIT/ussf-portal-cms.git#main
+      dockerfile: Dockerfile
+    restart: always
+    ports:
+      - '3001:3001'
+    environment:
+      - PORT=3001
+      - SESSION_SECRET='bHkJ6Js~=E~1(yw@a!hB`zFgA.QcaHbHkJ6Js~=E~1(yw@a!hB`zFgA.QcaH'
```

Comment:
I know it doesn't matter what this value is and that it has to be set to something but it makes me feel a little uneasy that it _looks_ like it might matter so what if we set it to something like `SESSION_SECRET='thisvaluecanbeanythingitisonlyforlocaldevelopment'`